src/vonage/errors.py

```diff
@@ -1,14 +1,14 @@
-class Error(Exception):
-    pass
-
-
-class ClientError(Error):
-    pass
-
-
-class ServerError(Error):
-    pass
-
-
-class AuthenticationError(ClientError):
-    pass
+class Error(Exception):

```

Comment:
Not sure why this is showing all changed, line ending differences again?

In either case, if this isn't doing anything, just nuke this file.

Reply:
Spacing and linting